### PR TITLE
Logos banners

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

File: src/components/ChangeImageButton.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  const onClick = loading
+    ? () => {}
+    : () => dispatch(uploadImage(uploadSettings))
+    .then(action => {
+      let { error, payload } = action
+      if (error) return
+      update(payload)
+    })
+
+  return <a className='edit-link' onClick={onClick}>{loading ? 'Loading...' : 'Change'}</a>
```
Comment (levity) [inline src/components/ChangeImageButton.js]: let's use the Cloud-Upload or Compose icons for this instead of text

**Comment by robbiecarlton** [comment]
screen shot 2017-01-25 at 1 52 54 pm

**Comment by levity** [comment]
I would follow Facebook's lead here and just have the icon itself, no
rounded border or background color.

**Comment by robbiecarlton** [comment]
screen shot 2017-01-25 at 2 41 22 pm

**Comment by robbiecarlton** [comment]
How's that?

**Comment by levity** [comment]
better! more left margin on the one over the banner, so that the top and
left margin are visually equal.

**Comment by robbiecarlton** [comment]
screen shot 2017-01-25 at 3 07 35 pm

**Review by levity — APPROVED** [review]

File: src/containers/person/PersonProfile.js [hunk 2]
```diff
@@ -136,9 +138,9 @@ const PersonProfile = compose(
     }
   }
 
-  return <CoverImagePage id='person' image={banner_url || defaultBanner}>
+  return <CoverImagePage id='person' image={banner_url || defaultBanner} person={isSelf ? currentUser : null}>
     <div className='opener'>
-      <NonLinkAvatar person={person} />
+      <NonLinkAvatar person={currentUser} showEdit={isSelf} />
```
Comment (levity) [inline src/containers/person/PersonProfile.js:143]: oops, look what i missed in review

**Pull request merged**